components/BottomSheet/src/MDCBottomSheetController.m

```diff
@@ -82,9 +82,11 @@ - (BOOL)accessibilityPerformEscape {
     return NO;
   }
   __weak __typeof(self) weakSelf = self;
-  [self dismissViewControllerAnimated:YES completion:^{
-    [weakSelf.delegate bottomSheetControllerDidDismissBottomSheet:weakSelf];
-  }];
+  [self
```

Comment:
It's a bit weird, but probably the only way to get the block to fit correctly. Since there's only one block argument, it can't use the exception mentioned in http://google.github.io/styleguide/objcguide.html#method-invocations .